project/scripts/bisect.scala

```diff
@@ -124,6 +125,7 @@ object ValidationScript:
 
   def tmpScalaCliScript(command: String, args: Seq[String]): File = tmpScript(s"""
     |#!/usr/bin/env bash
+    |export JAVA_HOME=${sys.props("java.home")}
```

Comment:
Any specific reason to override `JAVA_HOME` instead of passing the java version to scala-cli via `--jvm` option?
Also, this won't be taken into account when using a custom validation script. Is this intentional?

Reply:
I assume that user of custom validation script is an advanced user who understands not only a tested code, but also the environment in which it is being run. Especially since custom bisect script might use sbt or other build tool. We could add custom JAVA_HOME  to custom scripts by populating env variables of created process, but it would require a rewrite for parts of the code. I'd try this approach and check how it behaves.

The `JAVA_HOME` was used instead of `--jvm` argument to make sure that we exactly the same JVM version. We would not be able to pass a path in --jvm argument, and extraction of --jvm argument might be difficult.